tests/eas/acceptance.py

```diff
@@ -66,11 +67,20 @@ class EasTest(LisaTest):
         },
     }
 
+    # Set to true to run a test only on heterogeneous systems
+    skip_on_smp = False
```

Comment:
Since for the time being, all the tests we have are only for !smp systems, and actually most of the tests we will develop will be for !smp systems, should not be easier to set this default to True?